protected/humhub/modules/directory/views/directory/spaces.php

```diff
@@ -14,6 +14,12 @@
 use yii\helpers\Url;
 
 ?>
+
+<script>
```

Comment:
Normally you'd place script tags on the bottom line, but I'm kind of iffy on if more JavaScript should be added into the PHP code, HumHub already is heavily dependant on JavaScript as is. :thinking:

Reply:
In my case I had it deferred, so it's the same as placing it at the end and caused no problems.

Reply:
But I'm sure there is a way to do this without script tags. Because on that same page, there are working tooltips without the script that I added to initiate them.